CONTRIBUTING.rst

```diff
@@ -39,17 +39,17 @@ using a Git checkout:
 Now your local repo is set up such that you will push changes to your GitHub
 repo, from which you can submit a pull request.
 
-- Create a virtualenv in which to install gcloud-python::
+- Create a virtualenv in which to install ``gcloud-python``::
 
    $ cd ~/hack-on-gcloud
-   $ virtualenv -ppython2.7 env
+   $ virtualenv --python python2.7 env
```

Comment:
@tseaver Ditto here on unrelated changes. I was always like "WHAT is `-pp"? and have always been meaning to add the verbose flag.